content/maintainers-guide/how-to-become-a-committer.md

```diff
@@ -0,0 +1,61 @@
+---
+title: How to become a committer
+weight: 12
+---
+
+# How to become a committer
+
+In the PrestaShop project, committer are contributors with partial merge rights. 
+
+Committers participate in reviewing code contributions and are able to include them in the project once they have been accepted and verified.
+
```

Comment:
Please explain differences with maintainer. It's not clear.

Reply:
If I understand well, it is like B include A example 

<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/b0/Venn_A_subset_B.svg/1024px-Venn_A_subset_B.svg.png" width="20%" />

with A = being a committer and B = being a maintainer 
So every responsibility/power that can take a committer can be taken by a maintainer as well but not all responsibilities/powers of maintainers are given to committers.

Does this example make things clearer ?

Reply:
@Progi1984 Difference is in the approval rights :) this line is about "mission". Both committers and maintainers have the same mission about code review, not the same rights scope.

Just like I opened https://github.com/PrestaShop/open-source/issues/95 if you feel like there is a need for improvement, let's discuss it with the whole group 👍